Lab3/server/src/group/controller.ts

```diff
@@ -1,14 +1,35 @@
 import * as express from 'express';
+import * as uuid from 'uuid';
 import { GroupService } from './service';
 
 const app = express();
 
 export default ({ groupService }: { groupService: GroupService }) => {
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  app.use(express.json());
+  
   app.get('/', async (_, res) => {
     const groups = await groupService.table.select();
     res.json(groups).end();
   });
+  
+  app.post('/', async (req, res) => {
+    const group = req.body;
+    const sameGroups = await groupService.table.select(['*'], group);
+    if (!sameGroups.length) {
```

Comment:
We shouldn't do business logic inside controller. This should be moved to service (every similar occurence).